CHANGELOG.md

```diff
@@ -181,14 +198,72 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
 
 #### Core
 
-* We deprecated seqan3::range_compatible_concept and it will be removed in 3.1.0
+* Deprecated seqan3::range_compatible_concept and it will be removed in 3.1.0
   ([\#2265](https://github.com/seqan/seqan3/pull/2265)).
 
+Header Changes:
+
+```cpp
+#include <seqan3/core/algorithm/bound.hpp> [Functionality included in alignment/configuration/align_config_band.hpp]
+#include <seqan3/core/{algorithm => configuration}/configuration.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/concept.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_element_debug_mode.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_element_parallel_mode.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_utility.hpp>
+#include <seqan3/core/{algorithm => configuration}/pipeable_config_element.hpp>
```

Comment:
```suggestion
#include <seqan3/core/algorithm/bound.hpp> [Functionality included in alignment/configuration/align_config_band.hpp]
#include <seqan3/core/{algorithm => configuration}/configuration.hpp>
#include <seqan3/core/{algorithm => configuration}/pipeable_config_element.hpp>
```

* #include <seqan3/core/{algorithm => configuration/detail}/concept.hpp> was detail to begin with
* #include <seqan3/core/{algorithm => configuration/detail}/configuration_element_debug_mode.hpp> was detail to begin with
* #include <seqan3/core/{algorithm => configuration/detail}/configuration_element_parallel_mode.hpp> was detail to begin with
* #include <seqan3/core/{algorithm => configuration/detail}/configuration_utility.hpp> was detail to begin with